addpage.js

```diff
@@ -0,0 +1,46 @@
+const modal = document.getElementById("submitModal");
+const submitBtn = document.getElementById("submitBtn");
+
+const modalOptYes = document.getElementById("modalYes");
+const modalOptNo = document.getElementById("modalNo");
+
+const allInputs = document.getElementById("allInputs");
+
+//submit button functionality starts
+
+submitBtn.addEventListener("click", function () {
+    modal.style.display = "block";
+})
+
+modalOptYes.addEventListener("click", () => {
+    modal.style.display = "none";
+    allInputs.reset();
+})
+modalOptNo.addEventListener("click", () => {
+    modal.style.display = "none";
+})
+
+// submit button functionality ends
+
+// photo upload funcitonality starts
+
+const uploadPhotoBox = document.getElementById("uploadPhotoBox");
+const photoDisplayed = document.getElementById("photoDisplayed");
+console.log(photoDisplayed);
```

Comment:
Avoid pushing console.log , it is better only to use them during development locally .